pegasus/src/course.rb

```diff
@@ -3,9 +3,9 @@
 Lesson = Struct.new(:number, :url, :title, :description, :path)
 
 class Course
-  PRODUCTION_COURSES = %w(course1 course2 course3 course4 msm science misc algebra)
-  COURSES_WITHOUT_UNIT_NUMBERS = %w(course1 course2 course3 course4 msm algebra misc)
-  COURSES_WITH_PDF_GENERATION = %w(course1 course2 course3 course4 msm csp algebra misc)
+  PRODUCTION_COURSES = %w(course1 course2 course3 course4 msm science misc algebra advocacy)
```

Comment:
actually... do you even need to add advocacy here since you made a new route instead of reusing the curriculum/:kind ones?